migrations/upgrade-3-0-0.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Apply any database upgrades required for 3.0.0.
+ *
+ * Autoresponder configuration was moved from admin settings to widget settings.
+ * All widgets must be updated with an autoresponder field.
+ *
+ * @since 3.0.0
+ */
+function smailyforwp_upgrade_3_0_0() {
+	global $wpdb;
+	// Halt upgrade if no table exists.
```

Comment:
In accurate. Migration is not halted, merely autoresponders won't be migrated to widgets.

Reply:
Removed the comment altogether